lib/bsdlib/bsdlib.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <init.h>
+#include <device.h>
+#include <zephyr.h>
+
+#include <bsd.h>
+#include <bsd_platform.h>
+
+extern void ipc_proxy_irq_handler(void);
+
+static int _bsdlib_init(struct device *unused)
+{
+	/* Setup the network IRQ used by the BSD library.
+	 * Note: No enable irq_enable here, that is done through bsd_init.
```

Comment:
`No enable irq_enable here` sounds odd, perhaps you meant `No call to irq_enable here`?

Reply:
Updated old comment.